src/tribol/search/InterfacePairFinder.cpp

```diff
@@ -410,13 +407,14 @@ class GridSearch : public SearchBase {
     const auto mesh1 = m_coupling_scheme->getMesh1().getView();
     const auto mesh2 = m_coupling_scheme->getMesh2().getView();
     auto& contactPairs = m_coupling_scheme->getInterfacePairs();
+    auto binning_proximity_scale = m_coupling_scheme->getParameters().binning_proximity_scale;
 
     // Find matches in first mesh (with index 'fromIdx')
     // with candidate elements in second mesh (with index 'toIdx')
-    int k = 0;
+    // int k = 0;  // Debug only
```

Comment:
Just verify that this was only used for debug. Were we using this in any tests for output?

Reply:
No, it was only used in commented out debug output below.  I just commented this out to silence a warning.